examples/tutorials/core_api_pytorch_mnist/model_def_metrics.py

```diff
@@ -150,22 +174,27 @@ def main(core_context):
     )
     parser.add_argument("--seed", type=int, default=1, metavar="S", help="random seed (default: 1)")
 
+    ## Docs snippet start after: log interval
     # NEW: Change log interval to 100 to reduce network overhead.
     parser.add_argument(
         "--log-interval",
         type=int,
         default=100,
         metavar="N",
         help="how many batches to wait before logging training status",
+        # Docs snippet end before: log interval
```

Comment:
closing paren again.  I'm going to ask you to review all your doc-snippet-ends and I'll stop marking them.

Reply:
fixed